x-pack/plugins/cloud_security_posture/public/common/utils/get_dataset_display_name.ts

```diff
@@ -7,10 +7,12 @@
 
 type Dataset = 'wiz.cloud_configuration_finding' | 'cloud_security_posture.findings';
 
-export const WIZ_DATASET = 'wiz.cloud_configuration_finding';
 export const CSP_DATASET = 'cloud_security_posture.findings';
+export const CSP_VULN_DATASET = 'cloud_security_posture.vulnerabilities';
+export const WIZ_DATASET = 'wiz.cloud_configuration_finding';
+export const WIZ_VULN_DATASET = 'wiz.vulnerability';
 
 export const getDatasetDisplayName = (dataset?: Dataset | string) => {
```

Comment:
Yeah, only reason I didnt was because it was just temporary until we get `observer.provider` and just display the value. So it's just meant to be a temporary solution

Reply:
nothing is more constant than things intended to be temporary :) but your call, the logic is quite trivial